src/tests/checkers/phpdoc_test.go

```diff
@@ -425,3 +425,41 @@ func TestPHPDocIncorrectSyntaxOptionalTypesType(t *testing.T) {
 	}
 	test.RunAndMatch()
 }
+
+func TestPHPDocInvalidBeginning(t *testing.T) {
```

Comment:
Could you add tests that make sure that a single line phpdocs (not only `@var`) do not cause any warnings?